api/api.go

```diff
@@ -47,6 +47,14 @@ type API interface {
 	GetMetricsForTag(tagKey, tagValue string) ([]MetricKey, error)
 }
 
+type GraphiteStore interface {
```

Comment:
let's just make this into the `api/API` and figure out splitting it up later. I think we have very little things to gain by doing it (right now).